### DISCUSSION TITLE
Update Home Site

### DISCUSSION THREAD

**Discussion by duynhne** [dewanakl/undangan#22] [Ideas]
Hi 

Thank you for creating such a wonderful wedding page.
Can I suggest a Home page?
On the Home page, I can put the wallpaper of 2 people like the attached photo. Instead, it refers to the background color.
<img width="1386" alt="Screenshot 2024-04-25 at 16 29 13" src="https://github.com/dewanakl/undangan/assets/108656040/ed49816c-72ab-420c-b31e-cd67e561d80a">

Thank you bro

**Comment by dewanakl** [comment]
Hi
Thank you
you can fully customize this "undangan" invitation and deploy our self, thanks bro
